### ISSUE TITLE
Override HamburguerMenu header height

### ISSUE THREAD

**Issue by Albvadi** [MahApps/MahApps.Metro#3727]
Hi,
I wish I could add space between the header and the HamburguerMenu items.
![image](https://user-images.githubusercontent.com/6285255/72978837-6fbad680-3dd7-11ea-9b8a-d2367368370b.png)

I tried to add margin to the HamburgerMenuHeaderTemplate but the height it´s fixed and the header is cut.
```xaml
<Controls:HamburgerMenu.HamburgerMenuHeaderTemplate>
	<DataTemplate>
		<TextBlock HorizontalAlignment="Center" 
                   Margin="15" // <- Add this
                   FontSize="20"
                   FontWeight="Bold"
                   Text="MY HEADER TEXT"
        />
	</DataTemplate>
</Controls:HamburgerMenu.HamburgerMenuHeaderTemplate>
```
![image](https://user-images.githubusercontent.com/6285255/72979273-40f13000-3dd8-11ea-901b-b88aafa8c581.png)

It´s same as https://github.com/MahApps/MahApps.Metro/issues/3284, but adding my custom space...

Thanks!

**Comment by punker76** [comment]
@Albvadi If you already use 2.0 alpha then you can use [2 new properties](https://github.com/MahApps/MahApps.Metro/pull/3324) (PaneMargin and PaneHeaderMargin) to change this.

**Comment by Albvadi** [comment]
I´m using 1.6.5, because I´m totally newbie and all the examples and guides finded uses this version. Is it possible in this version?

It's a new application so I could update to 2.0 alpha, but the .NET Framework must be 4.5. 2.0 alpha works in 4.5 NET Framework?

**Comment by batzen** [comment]
The supported runtimes start with .net 4.5.2 and go up .net core 3.1.
So yes, .net 4.7 is covered.

**Comment by Albvadi** [comment]
> Just a question: If you're starting a new project why not try .net core 3.1 for that?

Basically out of fear.. :slightly_frowning_face:

I´m a web developer (PHP generally) and I have been commissioned this application for my company with around 12k computers with a Win10 custom base image which already has the 4.7 .NET version installed.
Due to my null knowledge in this C# world, actually I´m a copy-samples/stackoverflow developer... :sweat_smile: , so I search examples for what I need and adapt to my requirements...

I read a lot before starting the project (.Net core vs .Net Framework, Xaml better practices, MVVM patron, etc...) and because It´s not a complex application I think that I will not take advantage of the .Net core advantages, so I thought using more stable and long versions (1.6.5 mahApps.Metro and .Net Framework 4.7) would be much simpler for a newbie like me...

**Comment by punker76** [comment]
@Albvadi 

> I´m using 1.6.5, because I´m totally newbie and all the examples and guides finded uses this version. Is it possible in this version?
> 
> It's a new application so I could update to 2.0 alpha, but the .NET Framework must be 4.7. 2.0 alpha works in 4.7 NET Framework?

v2.0 alpha start with .net 4.5.2 and go up .net core 3.1. So yes, .Net 4.7 is covered.

**Comment by Albvadi** [comment]
@punker76 

I have updated the version of the framework from 1.6.5 to 2.0 alpha and after some changes, there are things that work correctly but there are others that do not and I´m not able to solve...

Do you prefer to speak a new issue to detail the problems with migration or do I do it in this same issue?

**Comment by timunie** [comment]
@Albvadi 
If you think it is no bug but support needed, we can see if we can help via gitter (https://gitter.im/MahApps/MahApps.Metro). 

Happy coding, 
Tim

**Comment by Albvadi** [comment]
> @Albvadi
> If you think it is no bug but support needed, we can see if we can help via gitter (https://gitter.im/MahApps/MahApps.Metro).
> 
> Happy coding,
> Tim

No, they are not bugs, I think it´s only I´m a newbie. I have solved some of my problems with the update, but surely I will need more help to carry out the application so I will go to Gitter in case you can help me out...

Thank you for all the help and support you give to this project. I wish all projects had this support!
